autotest/utilities/test_ogr2ogr.py

```diff
@@ -290,18 +290,55 @@ def test_ogr2ogr_13(ogr2ogr_path, tmp_path):
 
 def test_ogr2ogr_14(ogr2ogr_path, tmp_path):
 
-    output_shp = str(tmp_path / "poly.shp")
+    output_shp = tmp_path / "poly.shp"
 
-    gdaltest.runexternal(
+    # invalid value
+    _, err = gdaltest.runexternal_out_and_err(
+        ogr2ogr_path + f" -segmentize small_bits {output_shp} ../ogr/data/poly.shp poly"
+    )
+    assert "Failed to parse" in err
+    assert not output_shp.exists()
+
+    _, err = gdaltest.runexternal_out_and_err(
         ogr2ogr_path + f" -segmentize 100 {output_shp} ../ogr/data/poly.shp poly"
     )
 
+    assert not err
+
     ds = ogr.Open(output_shp)
     assert ds is not None and ds.GetLayer(0).GetFeatureCount() == 10
     feat = ds.GetLayer(0).GetNextFeature()
     assert feat.GetGeometryRef().GetGeometryRef(0).GetPointCount() == 36
 
 
+###############################################################################
+# Test -simplify
+
+
+@pytest.mark.require_geos
```

Comment:
For performance reasons, I would suggest to avoid adding more tests in test_foo.py when it is possible to have them in test_foo_lib.py  (forking a process is way slower than in-process execution of the utlity), which should be possible here

Reply:
Makes sense, I should have looked deeper when I didn't find a test for `-simplify` in this file already.